snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -309,8 +309,6 @@ func (b *bootstrapper) Notify(_ context.Context, msg common.Message) error {
 		)
 		return nil
 	}
-
-	b.Ctx.StateSyncing.Set(false)
```

Comment:
block backfilling may be enabled. We'll declare StateSyncing done when block backfilling is done